build/build-race-binaries.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Build cockroach binary with race detection enabled.
+
+set -euo pipefail
+
+source $(dirname $0)/build-common.sh
+
+# This is mildly tricky: This script runs itself recursively. The
+# first time it is run it does not take the if-branch below and
+# executes on the host computer. It uses the builder.sh script to run
+# itself inside of docker passing "docker" as the argument causing the
+# commands in the if-branch to be executed within the docker
+# container.
+if [ "${1-}" = "docker" ]; then
+    time make STATIC=1 build GOFLAGS="-race"
+    check_static cockroach
+    strip -S cockroach
```

Comment:
ok, so `-s` is gone, and `-w` is only for debugging info, so instead of a 4x saving in binary size, we're at 10% saving (on linux).